static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx

```diff
@@ -57,7 +57,7 @@ export function SpanDescriptionCell({
 
 const NULL_DESCRIPTION = <span>&lt;null&gt;</span>;
 
-const WiderHovercard = styled(
+export const WiderHovercard = styled(
```

Comment:
starfish!!!

this could be extracted, it's not starfish specific. and/or width modifications built into hovercard itself.